packages/video_player/video_player_web/lib/src/video_player.dart

```diff
@@ -205,18 +205,29 @@ class VideoPlayer {
   /// Moves the playback head to a new `position`.
   ///
   /// `position` cannot be negative.
+  ///
+  /// Does nothing if the new position is the same as the current position.
   void seekTo(Duration position) {
     assert(!position.isNegative);
+    // Don't seek if video is already at position.
+    // As seeking when completed will trigger another completed event ('onEnded')
+    // to avoid potentially firing extra 'onEnded' events when the video is already over
```

Comment:
This needs a period at the end of the sentence, since comments should be properly punctuated sentences.